daemon/update_linux.go

```diff
@@ -28,5 +28,6 @@ func toContainerdResources(resources container.Resources) libcontainerd.Resource
 	}
 	r.MemoryReservation = uint64(resources.MemoryReservation)
 	r.KernelMemoryLimit = uint64(resources.KernelMemory)
+	r.PidsLimit = uint64(resources.PidsLimit)
```

Comment:
Need help with this part. From my findings, I need to add `PidsLimit` here to update the running containers (real world). But `libcontainered.Resources` has no attribute named `PidsLimit`. Hence, the build would fail for now.

I found that the libcontainered resources attributes are defined in (github.com/docker/containerd/api/grpc/types)[https://github.com/docker/docker/blob/master/vendor/github.com/docker/containerd/api/grpc/types/api.proto], but I'm not sure how to generate `api-pb.go` once I edit `api.proto`.

Can someone help me with this?